scp.py

```diff
@@ -445,7 +445,10 @@ def _recv_file(self, cmd):
                 # we have to make sure we don't read the final byte
                 if size - pos <= buff_size:
                     buff_size = size - pos
-                file_hdl.write(chan.recv(buff_size))
+		data = chan.recv(buff_size)
```

Comment:
It looks like you mixed spaces and tabs. I can fix that though.

Reply:
Sorry, I was using vim 😅